maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PrepareReleaseMojo.java

```diff
@@ -387,4 +403,27 @@ protected void prepareRelease( boolean generateReleasePoms )
         }
     }
 
+    private String resolveLineSeparator()
+    {
+        if ( lineSeparator  == null )
+        {
+            return System.getProperty( "line.separator" );
+        }
+
+        switch ( lineSeparator )
```

Comment:
it would be really great if there was a "same as source" option that sniffed the line endings in the current projects `pom.xml` and used that as the default rather than the jvm default.  this would fix MRELEASE-899 without having to make any changes, and should not affect any user who was not affected by MRELEASE-899 (or its inverse that \n would be used on *nix when a \r\n file was checked out) already.

Reply:
Hey, i think the "same as source" option is a good idea in general but not as the default option because it would change the current behavior.
Maybe I will implement it soon, but not in this request.

Reply:
I prefer not to have an option at all: Same as source always, implicit.

Reply:
Would it be an alternative for you if I implement the "same as souce" option and set it as the default?